pkg/webhookconfig/common.go

```diff
@@ -29,7 +29,7 @@ var (
 	policyRule   = admissionregistrationv1.Rule{
 		Resources:   []string{"clusterpolicies/*", "policies/*"},
 		APIGroups:   []string{"kyverno.io"},
-		APIVersions: []string{"v1"},
+		APIVersions: []string{"v1", "v2beta1"},
```

Comment:
Is this enough to get the validation code executed ?

Reply:
yes, I believe. I have tested few basic validate policies works for me.
I'm missing anything?

Reply:
I guess it is validated against v1 ?

Reply:
validation code doesn't know about `v2beta1` (https://github.com/kyverno/kyverno/blob/2c8641690039a4bfaea4dab6a2b73aa16a8e518b/pkg/webhooks/policy/handlers.go#L31-L53).
Should there be some `v2beta1` validation ?

Reply:
let me check once again.

Reply:
currently no